src/lennybot/check/docker_image_available.py

```diff
@@ -114,20 +114,25 @@ def _authenticate_on_registry(self, registry: str, authentication_header: WwwAut
             "access_type": "offline",
         }
 
-        if registry in self._container_config.registries.keys():
-            registry_data = self._container_config.registries[registry]
-            params["password"] = registry_data.password
-            params["username"] = registry_data.username
+        registry_data = self._container_config.registries[registry]
+        password = registry_data.password
+        username = registry_data.username
```

Comment:
Whats happens if the registry does not exist?

Reply:
then it ends up in this if:

`if registry not in self._container_config.registries.keys():
            logging.debug("Registry not found in config")
            response = requests.get(url)`